.github/pull_request_template.md

```diff
@@ -16,9 +17,21 @@ Please complete this section if any breaking changes have been made, otherwise d
 
 - [ ] I have assigned this PR to myself
 - [ ] I have added at least 1 reviewer
-- [ ] I have tested this code
-- [ ] I have updated the README and other relevant documents (guides...)
-- [ ] I have added sufficient documentation both in code, as well as in the READMEs
+- [ ] I have updated the official documentation
+- [ ] I have added sufficient documentation in code
+
+## Testing
+
+- [ ] I ran the official test suite
+- [ ] I ran custom tests
```

Comment:
Custom tests = Manual tests?

Perhaps better to add: I have tested this code change manually and written steps down in the `## Steps to reproduce` section

Reply:
I think it's a good idea to rename the wording to `I have tested this code change manually`, and just leave it at that, since the section below it should be removed if anybody checks this option

I would also rename the first option as well to something like `I have tested this code with the official test suite`

What do you think?